src/snowflake/snowpark/session.py

```diff
@@ -1836,12 +1836,18 @@ def sproc(self) -> StoredProcedureRegistration:
         """
         return self._sp_registration
 
-    def call(self, sproc_name: str, *args: Any) -> Any:
+    def call(
+        self,
+        sproc_name: str,
+        *args: Any,
+        statement_params: Optional[Dict[str, str]] = None,
```

Comment:
I know the other statement_params also use this type hints, but I think it should be `Optional[Dict[str, Any]]` 🤔

Reply:
Yes this sounds right. Updated.